chrome/content/zoteroocr.js

```diff
@@ -49,7 +62,7 @@ Zotero.OCR = new function() {
 			if (!(yield OS.File.exists(imageList))) {
 				try {
 					yield Zotero.Utilities.Internal.exec(pdfinfo, [pdf, base + '.info.txt']);
-					yield Zotero.Utilities.Internal.exec(pdftoppm, ['-png', '-r', 300, pdf, dir + '\\page' ]);
+					yield Zotero.Utilities.Internal.exec(pdftoppm, ['-png', '-r', 300, pdf, dir + png_prefix]);
```

Comment:
We can always use `+ '/page'`, because Windows also supports '/' as a path separator.